docs/user/cli.md

```diff
@@ -291,48 +344,58 @@ This command tries to get the password from the standard input. If it is not the
 
 **Usage:** `agama auth login`
 
+
+
 ## `agama auth logout`
 
 Deauthenticate by removing the token
 
 **Usage:** `agama auth logout`
 
+
+
 ## `agama auth show`
 
 Print the used token to the standard output
 
 **Usage:** `agama auth show`
 
+
+
 ## `agama download`
 
 Download file from given URL
 
 The purpose of this command is to download files using AutoYaST supported schemas (e.g. device:// or relurl://). It can be used to download additional scripts, configuration files and so on. You can use it for downloading Agama autoinstallation profiles. However, unless you need additional processing, the "agama profile import" is recommended. If you want to convert an AutoYaST profile, use "agama profile autoyast".
 
-**Usage:** `agama download <URL>`
+**Usage:** `agama download <URL> <DESTINATION>`
 
 ###### **Arguments:**
 
-- `<URL>` — URL pointing to file for download
+* `<URL>` — URL pointing to file for download
+* `<DESTINATION>` — File name
+
+
 
 ## `agama finish`
 
-Finish the installation rebooting the system by default
+Finish the installation
 
 **Usage:** `agama finish [METHOD]`
 
 ###### **Arguments:**
 
-- `<METHOD>` — Action to be done at the end of the installation
+* `<METHOD>` — What to do after finishing the installation. Possible values:
+
+   stop - do not reboot and the system continues running.
+
+   reboot - reboot into the installed system.
+
+   halt - halt the installed machine.
+
+   poweroff - poweroff the installed machine.
+
+  Default value: `reboot`
+
 
-Possible values:
 
-- `stop`:
-  Stop the installation at its end, at the "Congratulations! [Reboot]" screen,
-  leaving the backend running
```

Comment:
Sorry I object to your deletion of my text :smile: 

When you say "the system continues running" it is left unsaid what exactly continues running. IMHO it is useful to say that it is including the Agama backend